torch/quantization/observer.py

```diff
@@ -799,6 +802,129 @@ def forward(self, x_orig):
         return x_orig
 
 
+class InputWeightMinMaxObserver(PerChannelMinMaxObserver):
```

Comment:
Let us prefix this with `_` since we don't want to expose this to the user. It might also be a good idea to put this into a separate file with the rest of the equalization stuff.